homeassistant/components/media_player/plex.py

```diff
@@ -46,6 +49,10 @@
     cv.boolean,
     vol.Optional(CONF_USE_CUSTOM_ENTITY_IDS, default=False):
     cv.boolean,
+    vol.Optional(CONF_PURGE_UNAVAILABLE_CLIENTS, default=True):
+    cv.boolean,
+    vol.Optional(CONF_CLIENT_PURGE_INTERVAL, default=0):
```

Comment:
Use `cv.time_period`. That will return in a timedelta. Then use `value.total_seconds()` to get back to seconds.

Reply:
Oh, and use `vol.All(cv.time_period, cv.positive_timedelta))` to make sure it's a positive duration.